cfg/rationalize_test.go

```diff
@@ -371,3 +371,54 @@ func TestRationalize_WriteConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestRationalizeMetricsConfig(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		name     string
+		config   *Config
+		expected int64
+	}{
+		{
+			name: "both_0",
```

Comment:
Can we add one for both unset

Reply:
Both will be set to 0. So it's already covered.